src/field_player_avatar.c

```diff
@@ -1725,7 +1726,20 @@ static void Task_WaitStopSurfing(u8 taskId)
 
 #define FISHING_PROXIMITY_BOOST 4
 #define FISHING_STICKY_BOOST    36
-#define FISHING_DEFAULT_ODDS    50
+
+#if I_FISHING_BITE_ODDS >= GEN_4
+    #define FISHING_OLD_ROD_ODDS 75
+    #define FISHING_GOOD_ROD_ODDS 50
+    #define FISHING_SUPER_ROD_ODDS 25
+#elif I_FISHING_BITE_ODDS >= GEN_3
+    #define FISHING_OLD_ROD_ODDS 50
+    #define FISHING_GOOD_ROD_ODDS 50
+    #define FISHING_SUPER_ROD_ODDS 50
+#elif I_FISHING_BITE_ODDS >= GEN_1
```

Comment:
Shouldn't this just be ``#else``? Otherwise we'll probably get build errors if `I_FISHING_BITE_ODDS` is `< GEN_1` for whatever reason

Reply:
The all 50 one should be the default, but then I'd have to duplicate it after gen 1 I believe

Reply:
I don't really think we should care about it if someone sets it to less than GEN_1, they're doing something wrong anyways

Reply:
what is the problem with just using `else` at the end?

Reply:
Yeah I'll just do that